<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileProtect</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTreeView" name="treeView">
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="currentRoot">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item row="0" column="7">
        <widget class="QPushButton" name="bnF7ShowDecrypted">
         <property name="text">
          <string>F7 Show decrypted</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="bnF5Encrypt">
         <property name="text">
          <string>F5 Encrypt</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="bnF6Decrypt">
         <property name="text">
          <string>F6 Decrypt</string>
         </property>
        </widget>
       </item>
       <item row="0" column="10">
        <widget class="QPushButton" name="bnF8Wipe">
         <property name="text">
          <string>F8 Wipe</string>
         </property>
        </widget>
       </item>
       <item row="0" column="11">
        <widget class="QPushButton" name="bnF9Options">
         <property name="text">
          <string>F9 Options</string>
         </property>
        </widget>
       </item>
       <item row="0" column="12">
        <widget class="QPushButton" name="bnCtrlPgUpGoUp">
         <property name="text">
          <string>Ctrl + PgUp Go up</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="bnF2SetAsRoot">
         <property name="text">
          <string>F2 Set as root</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="bnF3View">
         <property name="text">
          <string>F3 View</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="bnF4Execute">
         <property name="text">
          <string>F4 Execute</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>995</width>
     <height>20</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionUp_one_level"/>
    <addaction name="actionSet_as_root"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionShow_decrypted"/>
    <addaction name="separator"/>
    <addaction name="actionEncrypt_selected"/>
    <addaction name="actionDecrypt_selected"/>
    <addaction name="separator"/>
    <addaction name="actionWipe_selected"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUp_one_level">
   <property name="text">
    <string>Up one level</string>
   </property>
  </action>
  <action name="actionSet_as_root">
   <property name="text">
    <string>Set as root</string>
   </property>
  </action>
  <action name="actionEncrypt_selected">
   <property name="text">
    <string>Encrypt selected</string>
   </property>
  </action>
  <action name="actionDecrypt_selected">
   <property name="text">
    <string>Decrypt selected</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options...</string>
   </property>
  </action>
  <action name="actionWipe_selected">
   <property name="text">
    <string>Wipe selected</string>
   </property>
  </action>
  <action name="actionShow_decrypted">
   <property name="text">
    <string>Show decrypted...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
